bridger/clubhouse/__init__.py

```diff
@@ -7,11 +7,15 @@
 from rest_framework.response import Response
```

Comment:
This can actually be deleted, we do not use it anymore. This just for the next change. For now it is fine.